STL_Extension/include/CGAL/STL_Extension/internal/mesh_option_classes.h

```diff
@@ -165,6 +168,116 @@ struct Features_options
   bool b_;
 };
 
+template <typename Value>
+class Input_const_iterator_interface
```

Comment:
Can you explain what this class and the following ones are about?

Actually, there should be documenting comments in the code: not comments to be processed by Doxygen, but comments for readers of the code.